pyjobs/core/views.py

```diff
@@ -334,6 +334,49 @@ def job_application_challenge_submission(request, pk):
     )
 
 
+@staff_member_required
+def applied_users_details(request, pk):
+    def bool_to_string(flag):
+        return "Sim" if flag else "Não"
+
+    headers = {
```

Comment:
It doesn't have sense stating the headers of the table on two files.
Declare then on the HTML file only.